6_visualize/src/create_gif_tasks.R

```diff
@@ -24,6 +25,29 @@ create_gif_tasks <- function(timestep_ind, folders, view_ind, basemap_ind, storm
     paste(strs, collapse=sep)
   }
 
+  point_frame <- scipiper::create_task_step(
+    step_name = 'point_frame',
+    target_name = function(task_name, step_name, ...){
+      cur_task <- dplyr::filter(rename(tasks, tn=task_name), tn==task_name)
+      sprintf('6_vizprep/out/storm_point_[%s].rds.ind', cur_task$date_hour)
```

Comment:
For these point frames, let's cut straight to the data file (`.rds` only) rather than adding the shared-cache layer (`.rds.ind`). 

The benefit of just writing a data file, and not posting to google drive, is that we save a little bit of file writing time and a lot of file posting/getting time.

The drawback is that we will not be sharing these .rds files across the team. Instead, what this means is that anybody who wants to see these .rds files, or anything that depends on them, will need to build them themselves. This constraint will also apply to files that depend on these .rds files - in particular, the .png files will need to follow a similar pattern (.png only, no .png.ind files). But I think that's OK and what we want for this project.

Reply:
Sounds good. I think that is as simple as just cutting out that `.ind` step here?

Reply:
yep, pretty much, and skipping the call to gd_put